awx/api/views/__init__.py

```diff
@@ -371,13 +371,38 @@ def get_queryset(self):
 
 
 class InstancePeersList(SubListAPIView):
-    name = _("Instance Peers")
+    name = _("Peers")
     parent_model = models.Instance
-    model = models.Instance
-    serializer_class = serializers.InstanceSerializer
+    model = models.ReceptorAddress
+    serializer_class = serializers.ReceptorAddressSerializer
     parent_access = 'read'
-    search_fields = {'hostname'}
     relationship = 'peers'
+    search_fields = 'address'
+
+
+class InstanceReceptorAddressesList(ListCreateAPIView):
```

Comment:
should this be a `SubListCreateAPIView`?

It looks like you're trying to create a list based on a reverse ForeignKey relationship. I don't love that when used for creation, but it's done here-and-there, and this is as valid of a case as any. This type of view needs the `parent_key` set (I assume `instance`).